### PR TITLE
bump alpine to 3.19

### PR DESCRIPTION
**Pull request by esigo**

### CODE REVIEW

File: cpp_format_tools/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ARG BUILDIFIER_VERSION=3.5.0
-ARG CLANG_VERSION=10.0.1-r0
+ARG CLANG_VERSION=17.0.5-r0
```
Comment (marcalff) [inline cpp_format_tools/Dockerfile:2]: If we use clang 17, will the result of clang-format be the same as currently with version 10 ?

Comment (lalitb) [inline cpp_format_tools/Dockerfile:2]: I am not fully familiar with the purpose of this docker image. But if it is meant to format otel-cpp, as of now we need clang-format v10 for formatting. Will this upgrade bump up the clang format too?

Comment (lalitb) [reply cpp_format_tools/Dockerfile:2]: oh same question by @marcalff  :)

Comment (marcalff) [reply cpp_format_tools/Dockerfile:2]: Resolving, we since decided to upgrade clang-format anyway.

**Review by marcalff — APPROVED** [review]
LGTM

**Pull request merged**